finding/finding.go

```diff
@@ -102,13 +203,22 @@ func (f *Finding) WithMessage(text string) *Finding {
 // No copy is made.
 func (f *Finding) WithLocation(loc *Location) *Finding {
 	f.Location = loc
+	if f.Remediation != nil && loc != nil {
```

Comment:
is there any ordering to be worried about here?

In terms of when remediation info is set vs when location is?

Reply:
I've updated the code to use `f.Location` instead of `loc` in the part that sets the remediation. Thanks